Makefile

```diff
@@ -5,6 +5,8 @@ USER ?= $(shell id -un)
 UID ?= $(shell id -u)
 IP := $(firstword $(shell ip addr show | grep -Eo 'inet (addr:)?([0-9]*\.){3}[0-9]*' | grep -Eo '([0-9]*\.){3}[0-9]*' | grep -v '127.0.0.1' | awk '{print $1}' ))
 OS_NAME := $(shell uname -s | tr A-Z a-z)
+LOCALIP=$(shell ip -4 address show eth0 | grep -Eo 'inet (addr:)?([0-9]*\.){3}[0-9]*' | grep -Eo '([0-9]*\.){3}[0-9]*')
+AWS_NITRO_CLI_REVISION = dff9102783959412dcf5d515e641c2c3ad0d443b
```

Comment:
Is this particular revision important?

Reply:
Not particularly. But I added the specific version at @nspin 's suggestion to make builds repeatable (which is a Good Idea).

Reply:
OK, that makes sense.